plugins/Stk/Mallets/Mallets.h

```diff
@@ -255,6 +256,7 @@ public slots:
 
 	ComboBox * m_presetsCombo;
 	Knob * m_spreadKnob;
+	Knob * m_randomKnob;
```

Comment:
```suggestion
	Knob* m_randomKnob;
```

Reply:
I couldn't find anything about this in the coding conventions.

Reply:
Sorry, I'll add that these are just suggestions I would enact personally. I tend to follow a certain format, which puts the star right next to the type. If anything, feel free to resolve/ignore this if necessary.